<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="ROC-RDEVWS10" name="com.NewGIDirect.TC_085_DocuSign" tests="1" failures="0" timestamp="26 Sep 2018 05:47:31 GMT" time="47.482" errors="1">
  <testcase name="createQAA" time="47.482" classname="com.NewGIDirect.TC_085_DocuSign">
    <error type="org.sikuli.script.FindFailed" message="C://Work8Aug2018//directgl//Testing//BTIS_Application//LegacyApplication//SikuliPngs//Iagree.PNG: (317x24) in S(0)[0,0 1366x768] E:Y, T:3.0">
      <![CDATA[FindFailed: C://Work8Aug2018//directgl//Testing//BTIS_Application//LegacyApplication//SikuliPngs//Iagree.PNG: (317x24) in S(0)[0,0 1366x768] E:Y, T:3.0
  Line 2761, in file Region.java

at org.sikuli.script.Region.wait(Region.java:2761)
at org.sikuli.script.Region.find(Region.java:2326)
at org.sikuli.script.Region.getLocationFromTarget(Region.java:3220)
at org.sikuli.script.Region.click(Region.java:3718)
at org.sikuli.script.Region.click(Region.java:3703)
at com.BtisNewUI.PageMethods.GlDirectMethods.docuSign1(GlDirectMethods.java:256)
at com.NewGIDirect.TC_085_DocuSign.createQAA(TC_085_DocuSign.java:23)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- createQAA -->
</testsuite> <!-- com.NewGIDirect.TC_085_DocuSign -->
